dataactbroker/handlers/fileHandler.py

```diff
@@ -648,3 +722,10 @@ def completeGeneration(self, generationId):
         if generationId is None:
             return JsonResponse.error(ResponseException("Must include a generation ID",StatusCode.CLIENT_ERROR), StatusCode.CLIENT_ERROR)
 
+        # Pull url from request
+        safeDictionary = RequestDictionary(self.request)
+        url =  safeDictionary.getValue("href")
+        #Pull information based on task key
+        task = self.interfaces.jobDb.session.query(FileGenerationTask).options(joinedload(FileGenerationTask.file_type)).filter(FileGenerationTask.generation_task_key == generationId).one()
```

Comment:
We do specify the relationship in the model, but those relationships are not populated unless this type of load is done.  Since the next lines use the file_type.name, this was simpler than doing another lookup to load the file_type name based on file_type_id.